app/utils/IncidentP5Helper.scala

```diff
@@ -78,45 +79,49 @@ class IncidentP5Helper(
     ).flatten
   )
 
-  def endorsementDateRow: Option[SummaryListRow] = buildRowFromAnswer[String](
-    answer = Some("endorsement date"), // TODO: Pull from incident data
-    formatAnswer = formatAsText,
-    prefix = "departure.notification.incident.index.endorsement",
-    id = None,
-    call = None
-  )
-
-  def authorityRow: Option[SummaryListRow] = buildRowFromAnswer[String](
-    answer = Some("authority"), // TODO: Pull from incident data
-    formatAnswer = formatAsText,
-    prefix = "departure.notification.incident.index.authority",
-    id = None,
-    call = None
-  )
-
-  def endorsementCountryRow: Option[SummaryListRow] = buildRowFromAnswer[String](
-    answer = Some("endorsementCountry"), // TODO: Pull from incident data
-    formatAnswer = formatAsText,
-    prefix = "departure.notification.incident.index.endorsementCountry",
-    id = None,
-    call = None
-  )
-
-  def locationRow: Option[SummaryListRow] = buildRowFromAnswer[String](
-    answer = Some("location"), // TODO: Pull from incident data
-    formatAnswer = formatAsText,
-    prefix = "departure.notification.incident.index.location",
-    id = None,
-    call = None
-  )
-
-  def endorsementSection: StaticSection = StaticSection(
+  def endorsementDateRow(incidentIndex: Index): Option[SummaryListRow] =
```

Comment:
Could the `incidentIndex` be a param in `theIncidentP5Helper` class?

Reply:
We also have a `isMultipleIncidents` value that doesn't appear to be being used? Do we know why that is there?

Reply:
Also if all of these methods are ultimately going to call `data.Consignment.Incident(incidentIndex.position)` then why not just change the class definition to have `incident: IncidentType03` instead of `data: CC182CType` ?